TestSidecar seems to be flaky

#### Version information:
```
* 1e8bf85 - (HEAD -> master, origin/master) codecov set informational flag to true (#324) (17 minutes ago) <Anton Evangelatov>
```

#### Description:

Test failed on one Travis runner and passed on another - for a PR that has not touched any sidecar code - https://github.com/ipfs/testground/pull/275 - and rebased on master on `1e8bf85`

Will report if I see a similar test fail again.

```
=== RUN   TestSidecar
2020-01-07T16:34:23.152Z	INFO	client/client.go:31	testground client initialized	{"addr": "127.0.0.1:38565"}
2020-01-07T16:34:23.152Z	INFO	server/server.go:81	daemon listening	{"addr": "127.0.0.1:38565"}
Step 1/25 : ARG GO_VERSION=1.13.4
Step 2/25 : FROM golang:${GO_VERSION}-buster
 ---> ed081345a3da
Step 3/25 : ARG GO_IPFS_VERSION
 ---> Using cache
 ---> 53fc7df0f44b
Step 4/25 : ARG TESTPLAN_EXEC_PKG
 ---> Using cache
 ---> 6b863b60478d
Step 5/25 : ARG GO_PROXY=direct
 ---> Using cache
 ---> 6b1b152d21cc
Step 6/25 : ENV TESTPLAN_EXEC_PKG ${TESTPLAN_EXEC_PKG}
 ---> Using cache
 ---> 668cff757454
Step 7/25 : ENV PLAN_DIR /plan/
 ---> Using cache
 ---> ec2b6e14b05c
Step 8/25 : RUN if [ -n "${GO_IPFS_VERSION}" ]; then echo Install IPFS ${GO_IPFS_VERSION} && cd /tmp && wget https://dist.ipfs.io/go-ipfs/v${GO_IPFS_VERSION}/go-ipfs_v${GO_IPFS_VERSION}_linux-amd64.tar.gz && tar xf go-ipfs_v${GO_IPFS_VERSION}_linux-amd64.tar.gz; fi
 ---> Using cache
 ---> bbf0d83c8053
Step 9/25 : RUN touch /tmp/delete_me
 ---> Using cache
 ---> 96008c84c00d
Step 10/25 : COPY /plan/go.mod ${PLAN_DIR}
 ---> 2c1af6d4f88f
Step 11/25 : COPY /sdk/sync/go.mod /sdk/sync/go.mod
 ---> 4a8cf5a79635
Step 12/25 : COPY /sdk/iptb/go.mod /sdk/iptb/go.mod
 ---> 7f3dfe02d663
Step 13/25 : COPY /sdk/runtime/go.mod /sdk/runtime/go.mod
 ---> 874ecb464fa9
Step 14/25 : RUN cd ${PLAN_DIR}     && go env -w GOPROXY="${GO_PROXY}"     && go mod download
 ---> Running in 4b5bf15fe287
go: finding cloud.google.com/go v0.26.0
go: finding github.com/BurntSushi/toml v0.3.1
go: finding github.com/aead/siphash v1.0.1
go: finding github.com/btcsuite/btcd v0.0.0-20190824003749-130ea5bddde3
go: finding github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
go: finding github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
go: finding github.com/btcsuite/go-socks v0.0.0-20170105172521-4720035b7bfd
go: finding github.com/btcsuite/goleveldb v0.0.0-20160330041536-7834afc9e8cd
go: finding github.com/btcsuite/snappy-go v0.0.0-20151229074030-0bdef8d06723
go: finding github.com/btcsuite/websocket v0.0.0-20150119174127-31079b680792
go: finding github.com/btcsuite/winsvc v1.0.0
go: finding github.com/client9/misspell v0.3.4
91mgo: finding github.com/coreos/go-semver v0.3.0
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/dustin/go-humanize v1.0.0
go: finding github.com/fsnotify/fsnotify v1.4.7
go: finding github.com/go-redis/redis/v7 v7.0.0-beta.4
go: finding github.com/gogo/protobuf v1.3.0
go: finding github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
go: finding github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6
go: finding github.com/golang/mock v1.1.1
go: finding github.com/golang/protobuf v1.3.1
go: finding github.com/google/go-cmp v0.3.0
go: finding github.com/gxed/hashland/keccakpg v0.0.1
go: finding github.com/gxed/hashland/murmur3 v0.0.1
go: finding github.com/hashicorp/errwrap v1.0.0
go: finding github.com/hashicorp/go-multierror v1.0.0
go: finding github.com/hpcloud/tail v1.0.0
go: finding github.com/ipfs/go-cid v0.0.3
go: finding github.com/ipfs/go-ipfs-files v0.0.6
go: finding github.com/jbenet/go-cienv v0.1.0
go: finding github.com/jbenet/goprocess v0.1.3
go: finding github.com/jessevdk/go-flags v1.4.0
go: finding github.com/jrick/logrotate v1.0.0
go: finding github.com/kami-zh/go-capturer v0.0.0-20171211120116-e492ea43421d
go: finding github.com/kisielk/errcheck v1.2.0
go: finding github.com/kisielk/gotool v1.0.0
go: finding github.com/kkdai/bstream v0.0.0-20161212061736-f391b8402d23
go: finding github.com/konsorten/go-windows-terminal-sequences v1.0.1
go: finding github.com/kr/pretty v0.1.0
go: finding github.com/kr/pty v1.1.1
go: finding github.com/kr/text v0.1.0
go: finding github.com/libp2p/go-buffer-pool v0.0.1
go: finding github.com/libp2p/go-flow-metrics v0.0.1
go: finding github.com/libp2p/go-libp2p-core v0.2.3
go: finding github.com/libp2p/go-msgio v0.0.4
go: finding github.com/libp2p/go-openssl v0.0.2
go: finding github.com/mailru/easyjson v0.0.0-20180823135443-60711f1a8329
go: finding github.com/mattn/go-colorable v0.0.9
go: finding github.com/mattn/go-isatty v0.0.4
go: finding github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: finding github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1
go: finding github.com/minio/sha256-simd v0.1.1
go: finding github.com/mr-tron/base58 v1.1.2
go: finding github.com/multiformats/go-base32 v0.0.3
go: finding github.com/multiformats/go-multiaddr v0.1.1
go: finding github.com/multiformats/go-multibase v0.0.1
go: finding github.com/multiformats/go-multihash v0.0.8
go: finding github.com/onsi/ginkgo v1.8.0
go: finding github.com/onsi/gomega v1.5.0
go: finding github.com/pkg/errors v0.8.1
go: finding github.com/pmezard/go-difflib v1.0.0
go: finding github.com/sirupsen/logrus v1.2.0
go: finding github.com/smola/gocompat v0.2.0
go: finding github.com/spacemonkeygo/spacelog v0.0.0-20180420211403-2296661a0572
go: finding github.com/spaolacci/murmur3 v1.1.0
go: finding github.com/src-d/envconfig v1.0.0
go: finding github.com/stretchr/objx v0.1.1
go: finding github.com/stretchr/testify v1.4.0
go: finding github.com/x-cray/logrus-prefixed-formatter v0.5.2
go: finding go.opencensus.io v0.22.1
go: finding go.uber.org/atomic v1.4.0
go: finding go.uber.org/multierr v1.1.0
go: finding go.uber.org/zap v1.10.0
go: finding golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392
go: finding golang.org/x/exp v0.0.0-20190121172915-509febef88a4
go: finding golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3
go: finding golang.org/x/net v0.0.0-20190620200207-3b0461eec859
go: finding golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be
go: finding golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
go: finding golang.org/x/sys v0.0.0-20190922100055-0a153f010e69
go: finding golang.org/x/text v0.3.2
go: finding golang.org/x/tools v0.0.0-20190311212946-11955173bddd
go: finding google.golang.org/appengine v1.4.0
go: finding google.golang.org/genproto v0.0.0-20190425155659-357c62f0e4bb
go: finding google.golang.org/grpc v1.20.1
go: finding gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127
go: finding gopkg.in/fsnotify.v1 v1.4.7
go: finding gopkg.in/src-d/go-cli.v0 v0.0.0-20181105080154-d492247bbc0d
go: finding gopkg.in/src-d/go-log.v1 v1.0.1
go: finding gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: finding gopkg.in/yaml.v2 v2.2.2
go: finding honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099
 ---> 6a99b50e2e49
Step 15/25 : COPY . /
 ---> 5cc2f83dff4c
Step 16/25 : RUN cd ${PLAN_DIR}     && go env -w GOPROXY="${GO_PROXY}"     && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o testplan ${TESTPLAN_EXEC_PKG}
 ---> Running in c81487326bdc
 ---> 927c880779f0
Step 17/25 : RUN cd ${PLAN_DIR}   && go list -m all > /testground_dep_list
 ---> Running in 8adb22a99e6a
 ---> b7d5e12536f1
 ---> b7d5e12536f1
Step 18/25 : FROM debian
 ---> b5d2d9b1597b
Step 19/25 : COPY --from=0 /testground_dep_list /
 ---> e0cf75dca30b
Step 20/25 : COPY --from=0 /plan/testplan /
 ---> 4ced1a2ee1f5
Step 21/25 : RUN mkdir -p /usr/local/bin
 ---> Running in c0e3687a789f
 ---> d1affb866532
Step 22/25 : COPY --from=0 /tmp/delete_me /tmp/go-ipfs/ipfs* /usr/local/bin/
 ---> bc86db4800bd
Step 23/25 : RUN rm -f /usr/local/bin/delete_me
 ---> Running in f91eae5a41a0
 ---> 0dfbd34853b2
Step 24/25 : ENV PATH="/usr/local/bin:${PATH}"
 ---> Running in 076ea76b3d31
 ---> 0a163a491da2
Step 25/25 : ENTRYPOINT [ "/testplan" ]
 ---> Running in d961e059c7f2
 ---> 96b4fa0fc7fb
 ---> 96b4fa0fc7fb
Successfully built 96b4fa0fc7fb
Successfully tagged testground-network:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
Successfully tagged 1a4c78bb-a280-42e4-869f-aa696973d5c7:latest
2020-01-07T16:35:18.843Z	INFO	runner/local_docker.go:169	creating container	{"runner": "local:docker", "run_id": "40aa116d-c2be-43fb-a8eb-2f9f1caf53fd", "name": "tg-network-ping-pong-40aa116d-c2be-43fb-a8eb-2f9f1caf53fd-0"}
2020-01-07T16:35:18.906Z	INFO	runner/local_docker.go:169	creating container	{"runner": "local:docker", "run_id": "40aa116d-c2be-43fb-a8eb-2f9f1caf53fd", "name": "tg-network-ping-pong-40aa116d-c2be-43fb-a8eb-2f9f1caf53fd-1"}
2020-01-07T16:35:18.959Z	INFO	runner/local_docker.go:216	starting containers	{"runner": "local:docker", "run_id": "40aa116d-c2be-43fb-a8eb-2f9f1caf53fd", "count": 2}
2020-01-07T16:35:20.749Z	INFO	runner/local_docker.go:241	started containers	{"runner": "local:docker", "run_id": "40aa116d-c2be-43fb-a8eb-2f9f1caf53fd", "count": 2}
   0.4356s    MESSAGE << 042c60d2233a >> I am 1
   0.4943s    MESSAGE << 6923aca61396 >> I am 2
   1.3958s    MESSAGE << 6923aca61396 >> waiting until ready
   1.4959s    MESSAGE << 042c60d2233a >> waiting until ready
   1.4960s    MESSAGE << 042c60d2233a >> writing my id
   1.4960s    MESSAGE << 042c60d2233a >> reading their id
   1.5961s    MESSAGE << 6923aca61396 >> writing my id
   1.5962s    MESSAGE << 6923aca61396 >> reading their id
   1.5962s    MESSAGE << 6923aca61396 >> returning their id
   1.5962s    MESSAGE << 6923aca61396 >> reading my id
   1.6963s    MESSAGE << 042c60d2233a >> returning their id
   1.6964s    MESSAGE << 042c60d2233a >> reading my id
   1.6964s    MESSAGE << 042c60d2233a >> done
   1.6964s    MESSAGE << 042c60d2233a >> ping RTT was 200.394991ms [200ms, 205ms]
   1.7965s    MESSAGE << 6923aca61396 >> done
   1.7965s    MESSAGE << 6923aca61396 >> ping RTT was 200.407583ms [200ms, 205ms]
   2.2979s    MESSAGE << 6923aca61396 >> waiting until ready
   2.4478s    MESSAGE << 042c60d2233a >> waiting until ready
   2.4478s    MESSAGE << 042c60d2233a >> writing my id
   2.4479s    MESSAGE << 042c60d2233a >> reading their id
   2.4579s    MESSAGE << 6923aca61396 >> writing my id
   2.4580s    MESSAGE << 6923aca61396 >> reading their id
   2.4580s    MESSAGE << 6923aca61396 >> returning their id
   2.4580s    MESSAGE << 6923aca61396 >> reading my id
   2.4681s    MESSAGE << 042c60d2233a >> returning their id
   2.4681s    MESSAGE << 042c60d2233a >> reading my id
   2.4681s    MESSAGE << 042c60d2233a >> done
   2.4682s    MESSAGE << 042c60d2233a >> ping RTT was 20.310883ms [20ms, 25ms]
   2.4782s    MESSAGE << 6923aca61396 >> done
   2.4839s       FAIL << 6923aca61396 >> aborted expected an RTT between 20ms and 25ms, got 25.907669ms
  28.9284s       FAIL << 042c60d2233a >> aborted context deadline exceeded waiting on ping-pong-10; not enough elements, required: 2, got: 1
2020-01-07T16:35:49.769Z	INFO	runner/local_docker.go:278	deleting containers	{"runner": "local:docker", "run_id": "40aa116d-c2be-43fb-a8eb-2f9f1caf53fd", "ids": ["6923aca61396cdffd463c0c7c244d1fb23646bb8ebb993dd368f8bb1358f0def", "042c60d2233acab6ae3525492b5847dd57860554ab29a7b3e01648403dce8415"]}
2020-01-07T16:35:50.022Z	WARN	tgwriter/tgwriter.go:75	engine run error	{"ruid": "65c6e8e0", "err": "2 nodes failed"}
github.com/ipfs/testground/pkg/tgwriter.(*TgWriter).WriteError
	/home/travis/gopath/src/github.com/ipfs/testground/pkg/tgwriter/tgwriter.go:75
github.com/ipfs/testground/pkg/server.(*Server).runHandler
	/home/travis/gopath/src/github.com/ipfs/testground/pkg/server/run.go:42
github.com/ipfs/testground/pkg/server.loggingHandler.func1
	/home/travis/gopath/src/github.com/ipfs/testground/pkg/server/server.go:102
net/http.HandlerFunc.ServeHTTP
	/home/travis/.gimme/versions/go1.13.5.linux.amd64/src/net/http/server.go:2007
github.com/gorilla/mux.(*Router).ServeHTTP
	/home/travis/gopath/pkg/mod/github.com/gorilla/mux@v1.7.3/mux.go:212
net/http.serverHandler.ServeHTTP
	/home/travis/.gimme/versions/go1.13.5.linux.amd64/src/net/http/server.go:2802
net/http.(*conn).serve
	/home/travis/.gimme/versions/go1.13.5.linux.amd64/src/net/http/server.go:1890
--- FAIL: TestSidecar (86.87s)
FAIL
coverage: [no statements]
FAIL	github.com/ipfs/testground/cmd/itest	208.040s
```

--- a/plans/network/main.go
+++ b/plans/network/main.go
@@ -255,7 +255,7 @@ func main() {
 		return
 	}
 
-	err = pingPong("10", 20*time.Millisecond, 25*time.Millisecond)
+	err = pingPong("10", 20*time.Millisecond, 30*time.Millisecond)
 	if err != nil {
 		runenv.Abort(err)
 		return

